### ISSUE TITLE
[dev] Problem with "full" version of dev

### ISSUE THREAD

**Issue by lionel-bijaoui** [icebob/vue-form-generator#115]
When I launch the latest [a69e244] dev environment, and go to "full" I get this error in the browser console and nothing show up
```
[Vue warn]: Cannot find element: #app 
```

I also get this in the console:
```
WARNING in ./~/moment/min/moment.min.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./locale in C:\Users\maria\dev\vue-form-generator\node_modules\moment\min
```
Which is certainly linked with the recent changes #113 & #114

**Comment by icebob** [comment]
Yes, it's a new issue, because I created 2 dev example. But webpack can't load the full demo (don't load the index.html, instead generate an empty html file).
But if you add index.html to the url, it'll work: http://localhost:8080/full/index.html
I don't know what is the problem, because the `multiselect` demo is working.

**Comment by cristijora** [comment]
I had same issue before making those changes.

**Comment by icebob** [comment]
The `moment.min.js` warning is come with #113. Because the min file contains `require`, what webpack try to resolve.

**Comment by lionel-bijaoui** [comment]
Ok, I'm not the only one 😅
I'm able to make  http://localhost:8080/full/ work

**Comment by icebob** [comment]
Will be resolved in #119
